modules/dgkeywordRtdProvider.js

```diff
@@ -25,6 +25,15 @@ export function getDgKeywordsAndSet(reqBidsConfigObj, callback, moduleConfig, us
   const timeout = (moduleConfig && moduleConfig.params && moduleConfig.params.timeout && Number(moduleConfig.params.timeout) > 0) ? Number(moduleConfig.params.timeout) : PROFILE_TIMEOUT_MS;
   const url = (moduleConfig && moduleConfig.params && moduleConfig.params.url) ? moduleConfig.params.url : URL + encodeURIComponent(window.location.href);
   const adUnits = reqBidsConfigObj.adUnits || getGlobal().adUnits;
+  callback = (function(cb) {
+    let done = false;
+    return function () {
+      if (!done) {
+        done = true;
+        return cb.apply(this, arguments);
+      }
+    }
+  })(callback);
```

Comment:
Should this change be included?

Reply:
When i merged in master in the last commit all the recent changes from master came in, I'll revert